mozilla/fx-private-relay-add-on
# Add Countdown Timer on Addon

Preview: 
<img width="402" alt="image" src="https://user-images.githubusercontent.com/13066134/187744859-c6485cdf-495f-4121-9555-977b2937f676.png">

Features:
- Adds the countdown timer (to be submitted to AMO + Chrome on the **6th of September**)
- Adds email trackers removed data to the dashboard statistics (available to both premium & non-premium)

**How to test:**
With a non-premium account, check that the popup panel displays the countdown timer instead of the tips view. 
With a premium account, check that the countdown timer is not visible to users. 
Set the date on your calendar to after the 27th of September, and then check that the non-premium view resets back to its default state.
